README.md

```diff
@@ -403,3 +403,47 @@ $ ./gradlew run --args="monitor --bootstrap.servers 192.168.103.39:9092"
 3. --prop.file: the path to a file that containing the properties to be passed to kafka admin.
 4. --topic: topics to track (default: track all non-synced partition by default)
 5. --track: keep track even if all the replicas are synced. Also attempts to discover any non-synced replicas. (default: false)
+
+## Kubernetes Cluster Quick Start
+
+The following scripts can build a kubernetes cluster.
+
+### Run Control Node
+
+The script creates a control plan of kubernetes, and also run yunikorn on it. 
+Run it on the node which you want to be the master node.
+```shell
+./docker/start_k8s.sh controlplan
+```
+After that, there will be an instruction of k8s to add a worker node.
+For example:
+```shell
+kubeadm join --discovery-token abcdef.1234567890abcdef --discovery-token-ca-cert-hash sha256:1234..cdef 1.2.3.4:6443
```

Comment:
It would be better to generate the command which can add worker directly. The k8s command is good but using our script can add more friendly helpers in the future.

Reply:
any feedback?